### PR TITLE
French update for latest strings

### PR DESCRIPTION
**Pull request by el00ruobuob**
Hi @m2049r & @erciccione,

This is the french par of #551.  
Let me know if anything is needed.

### CODE REVIEW

**Comment by el00ruobuob** [comment]
Hi @assylias,

Do you have a minute to review it?

File: app/src/main/res/values-fr/help.xml [hunk 1]
```diff
@@ -313,14 +313,14 @@
...
         <p>
-        1. Open the wallet you want to spend from<br>
-        2. Wait until the wallet is synced &amp; the "Give" button appears<br>
-        3. Touch the "Give" button
+        1. Ouvrez le portefeuille à partir duquel vous souhaitez envoyer des fonds<br>
+        2. Attendez que le portefeuille soit synchronisé &amp; que le bouton "Envoyer" apparaisse<br>
+        3. Taper sur le bouton "Envoyer"
         </p>
-        <p>The payment details will be filled in. Check them and proceed like for any other transaction.</p>
+        <p>Les détails du paiement seront remplis. Vérifiez-les et procédez de la même manière que pour tout autre transaction.</p>
```
Comment (assylias) [inline app/src/main/res/values-fr/help.xml]: tout autre -> toute

File: app/src/main/res/values-fr/strings.xml [hunk 2]
```diff
@@ -367,22 +367,22 @@
     <string name="node_nobookmark">%1$d meilleurs nœuds mis en favoris automatiquement</string>
     <string name="label_test">Tester</string><!--note: as in "Test a network connection"-->
 
-    <string name="send_address_resolve_bip70">Resolving Payment URI&#8230;</string>
-    <string name="send_address_not_bip70">Could not resolve Payment URI</string>
-    <string name="send_address_bip70">Resolved Payment URI &#x2714;</string>
+    <string name="send_address_resolve_bip70">Résolution de l\'URI de Paiement&#8230;</string>
```
Comment (assylias) [inline app/src/main/res/values-fr/strings.xml:370]: Not sure how this is used, but I would not put the P in Paiement in upper case. Same comment applies to lines below.

**Review by assylias — COMMENTED** [review]
Minor comments.

File: app/src/main/res/values-fr/strings.xml (hunk 2 above)
Comment (el00ruobuob) [reply app/src/main/res/values-fr/strings.xml:370]: I did put a upparcase P as the english version is using the upercase P. Not sure either.

File: app/src/main/res/values-fr/help.xml (hunk 1 above)
Comment (el00ruobuob) [reply app/src/main/res/values-fr/help.xml]: ok

File: app/src/main/res/values-fr/strings.xml (hunk 2 above)
Comment (el00ruobuob) [reply app/src/main/res/values-fr/strings.xml:370]: maybe @m2049r could tell us if this should be kept uppercase or not

**Comment by el00ruobuob** [comment]
Thank you @assylias!

**Review by erciccione — APPROVED** [review]
Thank you @el00ruobuob @assylias  :)

File: app/src/main/res/values-fr/strings.xml (hunk 2 above)
Comment (erciccione) [reply app/src/main/res/values-fr/strings.xml:370]: This depends by the language. English use a lot of upper case letters for stilistic reasons, but it's not like that in many other languages. Just do as you prefer, but keep it consistent :)

Comment (el00ruobuob) [reply app/src/main/res/values-fr/strings.xml:370]: As i did every string with upper case to keep it stilistic, i would keep it uppercase here too.  
But if @assylias wants, i could (later) pr a change to bring everything lowercase.

Comment (assylias) [reply app/src/main/res/values-fr/strings.xml:370]: You don’t typically use upper case in French except to start a sentence. But it’s not a big deal either. Your call!

**Review by m2049r — APPROVED** [review]

**Pull request merged**